waggle-dance-core/src/main/java/com/hotels/bdp/waggledance/mapping/service/impl/PrefixBasedDatabaseMappingService.java

```diff
@@ -257,7 +274,7 @@ public DatabaseMapping databaseMapping(@NotNull String databaseName) {
 
   private boolean isWhitelisted(String databasePrefix, String database) {
     Whitelist whitelist = mappedDbByPrefix.get(databasePrefix);
-    if ((whitelist == null) || whitelist.isEmpty()) {
```

Comment:
was this a bug before?

Reply:
Hmm I'm not sure, but if whitelist is empty now, we don't want to accept anything. Now, if whitelist is empty, it means that we need to reject everything.